microsoft-edge/devtools-guide-chromium/storage/cookies.md

```diff
@@ -58,43 +57,48 @@ The **Cookies** table contains the following fields:
 
 *  **Size**.  The size, in bytes, of the cookie.
 
-*  **HTTP**.  If true, this field indicates that the cookie should only be used over HTTP, and JavaScript modification isn't allowed.  See [HttpOnly cookies](https://developer.mozilla.org/docs/Web/HTTP/Cookies#Secure_and_HttpOnly_cookies).
+*  **HttpOnly**.  If true, this field indicates that the cookie should only be used over HTTP, and JavaScript modification isn't allowed.  See [HttpOnly cookies](https://developer.mozilla.org/docs/Web/HTTP/Cookies#Secure_and_HttpOnly_cookies).
 
 *  **Secure**.  If true, this field indicates that the cookie must be sent to the server only over a secure, HTTPS connection.  See [Secure cookies](https://developer.mozilla.org/docs/Web/HTTP/Cookies#Secure_and_HttpOnly_cookies).
 
 *  **SameSite**.  Contains `strict` or `lax` if the cookie is using the experimental [Samesite](https://developer.mozilla.org/docs/Web/HTTP/Cookies#SameSite_cookies) attribute.
 
-*  **Priority**.  Contains `low`, `medium` (default), or `high` if the cookie is using the deprecated [cookie Priority](https://bugs.chromium.org/p/chromium/issues/detail?id=232693) attribute.
+*  **SameParty**. 
+
+*  **Partition Key**.
+
+*  **Priority**.  Contains `low`, `medium` (default), or `high` if the cookie is using the deprecated [Cookie Priority](https://bugs.chromium.org/p/chromium/issues/detail?id=232693) attribute.
 
 
 <!-- ====================================================================== -->
 ## Filter cookies
 
 To filter cookies by **Name** or **Value**, use the **Filter** text box:
 
-![Filtering out any cookies that don't contain the text ID.](../media/storage-application-storage-cookies-filter-id.msft.png)
-
-Filtering by other fields isn't supported.
+![Filtering out any cookies that don't contain the text ID.](images/application-storage-cookies-filter-cookies.png)
 
+> [!NOTE]
```

Comment:
https://docs.microsoft.com/en-us/contribute/markdown-reference#alerts-note-tip-important-caution-warning
"Avoid notes, tips, and important boxes. Readers tend to skip over them. It's better to put that info directly into the article text."

Several alternatives:
h4 with a label/heading describing the point.
**Note:** blah

Reply:
@mikehoffms Got it, thanks

Reply:
@dan-wesley Pls close this "Outdated" conversation - been resolved.

Reply:
@dan-wesley (or @captainbrosset )  Please close this "Outdated" conversation as Resolved if this has been resolved in the latest files.